plugins/CorePluginsAdmin/angularjs/plugin-settings/plugin-settings.controller.js

```diff
@@ -66,9 +66,15 @@
                     }
 
                     $element.find('.confirm-password-modal').modal({ dismissible: false, onOpenEnd: function () {
-                        $('.modal.open #currentUserPassword').focus();
-                        $('.modal.open #currentUserPassword').off('keypress').keypress(onEnter);
-                    }}).modal('open');
+                            $('.modal.open #currentUserPassword').focus();
+                            $('.modal.open #currentUserPassword').off('keypress').keypress(onEnter);
+                            $('.pluginSettings input').prop("readonly",true);
+                            $('.confirm-password-modal input').prop("readonly",false);
+                        },
+                        onCloseStart: function () {
+                            $('.pluginSettings input').prop('readonly', false);
+                        },
+                    }).modal('open');
```

Comment:
I'm not sure if that will really solve the issue. I guess the `readonly` attributes are set to prevent password managers from filling in the field. On the one side, it might actually be too late to set that attribute when opening the modal. The HTML is already placed earlier and a password manager could already fill the field. On the other side I doubt that password managers will all respect the `readonly` attribute. There should also be a `autocomplete=off`, which is already ignore by certain password managers, so why should they respect other attributes?
I guess the cleanest solution would be to disconnect the password input field from angular and set the `passwordConfirmation` property to the input fields value once the modal is closed. That way the modal should always be shown, even if it is already filled.